app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
+        populate_template_with_trackler_data(File.read("./x/docs/md/track/BETTER.md")).gsub('TOPIC', topic_key.to_s.upcase)
```

Comment:
I'm leaning toward "don't worry so much about it," but I could totally be convinced otherwise.

The fallback content and the contributing instructions share a whole bunch of things in common -- in both cases, we're reading markdown files in the same directory and replacing some variables -- so I think it makes sense that their implementations share some things.